applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -497,7 +502,7 @@ return view.extend({
 					default: return ''; // bad value
 				}
 			}
-			return set_value;
```

Comment:
As code https://github.com/openwrt/luci/blob/b1968d9cdd6425821553138e0487df8840785afa/modules/luci-base/htdocs/luci-static/resources/form.js#L3681
the `set_value` will be `undefined`

Reply:
your code branch simplifies to:
```
if(set_value){
 something = set_value;
}
...
return something;
```

which is the same as:
```
if(set_value) return set_value;
return null;
```

How does this differ? What is the error you receive?

Reply:
Yes, that’s the problem.

The `cfgvalue` function should always return the selected value which isn’t `undefined`, so we need to cache it and return it later. (like the super method does) https://github.com/openwrt/luci/blob/b1968d9cdd6425821553138e0487df8840785afa/modules/luci-base/htdocs/luci-static/resources/form.js#L1826

The `set_value` will be passed in the `load` function.https://github.com/openwrt/luci/blob/b1968d9cdd6425821553138e0487df8840785afa/modules/luci-base/htdocs/luci-static/resources/form.js#L913
But in the later `render` function, the second parameter is undefined.https://github.com/openwrt/luci/blob/b1968d9cdd6425821553138e0487df8840785afa/modules/luci-base/htdocs/luci-static/resources/form.js#L3681

Issue:
Now the dropdown box always shows the undefined value even I have the right config `ec256`.
![image](https://github.com/user-attachments/assets/700edb0b-ff12-421a-a673-585e42383fab)

Reply:
Got it, the `super` method is a good way to resolve it.